content/blog/hpe-sustainability-insight-center-api-at-a-glance.md

```diff
@@ -0,0 +1,301 @@
+---
+title: HPE Sustainability Insight Center API at a glance
+date: 2024-06-24T12:44:18.079Z
+author: Didier Lalli
+authorimage: /img/didier-lalli.png
+disable: false
+tags:
+  - hpe-greenlake-api
+---
+<style>
+li {
+   font-size: 27px;
+   line-height: 33px;
+   max-width: none;
+}
+</style>
+
+HPE GreenLake provides a new service called HPE Sustainability Insight Center that can assist you in obtaining detailed information about carbon emissions, energy consumption and the cost of the infrastructure that is managed by HPE GreenLake. In this blog post, I will explain how to extract data from this service in a programmatic way using cURL and the HPE Sustainability Insight Center API.
+
+## What is HPE Sustainability Insight Center?
+
+HPE Sustainability Insight Center is a service that runs on HPE GreenLake. I can add it into my workspace from the HPE GreenLake catalogue under the Management & Governance category.
```

Comment:
catalogue (GB) vs catalog (USA)
Not sure if you want to change it to catalog.

Reply:
Ok